### ISSUE TITLE
better accessibility support - install accessibility tools by default in dom0 and templates

### ISSUE THREAD

**Issue by adrelanos** [QubesOS/qubes-issues#1596]
As agreed at 32c3, we want better accessibility support in Qubes by installing the  accessibility tools by default.

The following packages should be installed by default for better accessibility. ([Following advice from the Debian accessibility mailing list.](https://lists.debian.org/debian-accessibility/2015/12/msg00043.html))

gnome-orca espeakup brltty brltty-speechd brltty-x11 console-braille florence dasher kdeaccessibility kvkbd kmousetool kmag kmouth jovie xbrlapi festival qt-at-spi

I advice to implement this in form of a meta package. For Debian, it could look similar to this:

https://github.com/Whonix/anon-meta-packages/blob/dc2b97efef5af2ae67a4134480d1eb32f50da809/debian/control#L138-L148

Then we would add that meta package to  [qubes-builder-debian/blob/master/template_debian/packages_qubes.list](https://github.com/marmarek/qubes-builder-debian/blob/master/template_debian/packages_qubes.list).

In which Qubes package this meta package could be defined?

We need to install these accessibility tools in dom0 and all templates. The mouse related tools may be superfluous inside templates, because mouse pointer is provided by dom0. The screen reader tools must be installed inside all templates unfortunately, because to dom0, the window contents is just a bitmap and no text a sceen reader could read.

**Comment by tlaurion** [comment]
@marmarek?
